### PR TITLE
Access to File init

### PR DESCRIPTION
**Pull request by brightsider**

### CODE REVIEW

File: Pring/File.swift [hunk 1]
```diff
@@ -195,7 +195,7 @@ public final class File: NSObject {
         self.url = url
     }
 
-    internal convenience init?(property: [String: String]) {
```
Comment (1amageek) [inline Pring/File.swift]: Why should I expose this initialization method?
I've already prepared `convenience` .

File: Pring/SubCollection.swift [hunk 2]
```diff
@@ -257,6 +257,10 @@ extension SubCollection: Collection {
     public var first: T? {
         return _self.first
     }
+
```
Comment (1amageek) [inline Pring/SubCollection.swift]: Nice !!

File: Pring/File.swift (hunk 1 above)
Comment (1amageek) [reply Pring/File.swift]: This is a way of being used that is not supposed.
What is the reason for having information other than MetaData?

**Comment by 1amageek** [comment]
@brightsider 
Hi.
https://github.com/1amageek/Pring/blob/master/Pring/File.swift#L121

Files can now have additional data.

**Comment by 1amageek** [comment]
Thanks for your PR!!
I am looking forward to next PR

**Comment by brightsider** [comment]
@1amageek make init public, please

**Comment by 1amageek** [comment]
You are mistaking your usage.
Front and Back should be defined separately.

**Comment by 1amageek** [comment]
I have not disclosed the initializer to prevent such usage.

**Pull request closed without merging**